bin/bb/cmd/migrate.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/user"
+	"time"
+
+	"github.com/bytebase/bytebase/plugin/db"
+
+	// install mysql driver.
+	_ "github.com/bytebase/bytebase/plugin/db/mysql"
+	"github.com/spf13/cobra"
+)
+
+func newMigrateCmd() *cobra.Command {
+	var (
+		databaseType string
+		username     string
+		password     string
+		hostname     string
+		port         string
+		database     string
+		file         string
+		description  string
+		issueID      string
+
+		// SSL flags.
+		sslCA   string // server-ca.pem
+		sslCert string // client-cert.pem
+		sslKey  string // client-key.pem
+	)
+	migrateCmd := &cobra.Command{
+		Use:   "migrate",
+		Short: "Migrates the database schema",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			tlsCfg := db.TLSConfig{
+				SslCA:   sslCA,
+				SslCert: sslCert,
+				SslKey:  sslKey,
+			}
+			f, err := os.Open(file)
+			if err != nil {
+				return fmt.Errorf("failed to open sql file %s, got error: %w", file, err)
+			}
+			defer f.Close()
+			return migrateDatabase(context.Background(), databaseType, username, password, hostname, port, database, description, issueID, false /*createDatabase*/, f, tlsCfg)
+		}}
+
+	migrateCmd.Flags().StringVar(&databaseType, "type", "mysql", "Database type. (mysql or pg).")
+	migrateCmd.Flags().StringVar(&username, "username", "", "Username to login database. (default mysql:root pg:postgres).")
+	migrateCmd.Flags().StringVar(&password, "password", "", "Password to login database.")
+	migrateCmd.Flags().StringVar(&hostname, "hostname", "", "Hostname of database.")
```

Comment:
```suggestion
	migrateCmd.Flags().StringVar(&hostname, "hostname", "", "Database host.")
```

Reply:
host instead of hostname

both mysql and psql cli uses --host.

The idea is to follow the convention in mysql/psql, and if they don't agree, then use the best judgement (normally shorter one wins)